code/src/styles/MovieThumb.css

```diff
@@ -0,0 +1,32 @@
+.movie-thumb-wrapper {
+  color: #fff;
+  text-decoration: none;
+  position: relative;
+}
+
+.movie-thumb-wrapper .image-poster {
+  width: 100%;
+}
+
+.movie-thumb-wrapper .movie-text {
+  display: none;
```

Comment:
Screen readers generally ignore anything with display: none - just to keep in mind if this is important or not in this case